ios/RNShare.m

```diff
@@ -190,7 +208,7 @@ - (NSDictionary *)constantsToExport
     shareController.completionWithItemsHandler = ^(NSString *activityType, BOOL completed, __unused NSArray *returnedItems, NSError *activityError) {
         if (activityError) {
             failureCallback(activityError);
-        } else {
+        } else if (completed || activityType == nil) {
```

Comment:
could this be a fix for https://github.com/react-native-community/react-native-share/issues/624?
cc @felippepuhle